### ISSUE TITLE
Erigon service behind 1hour and 20 minutes + on average 

### ISSUE THREAD

**Issue by marbew** [ledgerwatch/erigon#5642]
[Current options passed to Erigon binary as service + prysm on the same vm]

**System information**
erigon version 2022.10.1-dev-3777b613

**OS & Version**
Ubuntu 20.04.1 LTS

ExecStart=/home/ubuntu/.local/bin/erigon
--db.pagesize="64kb" 
--chain="mainnet" 
--datadir="/home/ubuntu/.eth-mainnet-erigon/datadir/" 
--ethash.dagdir="/home/ubuntu/.erigon-eth-mainnet/ethash/" 
--http --http.addr="0.0.0.0"  
--port="30303" 
--http.port="8546" 
--http.compression 
--torrent.port="42069" 
--private.api.addr="0.0.0.0:9091" 
--http.vhosts="*" 
--http.corsdomain="*" 
--http.api="eth,debug,net,txpool,trace,web3,erigon" 
--ws 
--ws.compression 
--snapshots="true" 
--torrent.download.rate=20mb 
--metrics 
--metrics.addr="0.0.0.0" 
--metrics.port=6065 
--authrpc.jwtsecret=/home/ubuntu/key-hex/jwt.hex

Hey there beautiful people. Do you spot anything that should NOT be there as flag or a missing one in this service? 
What specific part of the log should revel important behaviour related to delays? There are no Warn or Error so far.
I can see constantly the Commit cycle with "Timings (slower than 50ms)"
[erigon.log.tar.gz](https://github.com/ledgerwatch/erigon/files/9723878/erigon.log.tar.gz)

The instance has: 4vCPU 30GB of RAM Storage on Raid 0 on gp3 EBS with 4000 IOPS each disk 

#### Actual behaviour
The delay was already there before the latest binary 2022.10.1, but after the merge. Before this release we experienced OOM.
No errors on logs attached apparently at the moment.

```
0101001@ethsync:~$ free -mh
               total        used        free      shared  buff/cache   available
Mem:            30Gi        26Gi       231Mi       1.0Mi       4.1Gi       3.9Gi
Swap:             0B          0B          0B
```

**Comment by marbew** [comment]
We updated our config to:
[Current options passed to Erigon binary as service + prysm on the same vm]

System information
erigon version 2.27.0

OS & Version
Ubuntu 20.04.1 LTS

ExecStart=/home/ubuntu/.local/bin/erigon

--chain="mainnet"
--datadir="/home/ubuntu/.eth-mainnet-erigon/datadir/"

--http --http.addr="0.0.0.0"
--port="30303"
--http.port="8546"
--http.compression
--torrent.port="42069"
--private.api.addr="0.0.0.0:9091"
--http.vhosts=""
--http.corsdomain=""
--http.api="eth,debug,net,txpool,trace,web3,erigon"
--ws
--ws.compression
--snapshots="true"

--metrics
--metrics.addr="0.0.0.0"
--metrics.port=6065
--authrpc.jwtsecret=/home/ubuntu/key-hex/jwt.hex

and re-sync is happening.

**Comment by marbew** [comment]
This has been solved by updating to erigon 2.28.1 and prysm 3.1.2 on aws vm with 20000 IOPS on 4 EBS volumes in RAID0.
We are now at the tip of chain constantly with about 10/12 seconds delay

`[Service]
  Restart=on-failure
  RestartSec=5s
  ExecStart=/home/ubuntu/.local/bin/prysm/prysm.sh beacon-chain --execution-endpoint=http://localhost:8551 --jwt-secret=/home/ubuntu/key-hex/jwt.hex --monitoring-host=0.0.0.0 --datadir=/home/ubuntu/.prysm/ --slots-per-archive-point=64  
  Environment= USE_PRYSM_VERSION=v3.1.2
  Type=simple
  User=ubuntu`

`[Service]
  Restart=on-failure
  RestartSec=5s
  ExecStart=/home/ubuntu/.local/bin/erigon --chain="mainnet" --datadir="/home/ubuntu/.eth-mainnet-erigon/datadir/" --http --http.addr="0.0.0.0"  --port="30303" --http.port="8546" --http.compression --torrent.port="42069" --private.api.addr="0.0.0.0:9091" --http.vhosts="*" --http.corsdomain="*" --http.api="eth,debug,net,txpool,trace,web3,erigon" --ws --ws.compression --snapshots="true" --metrics --metrics.addr="0.0.0.0" --metrics.port=6065 --authrpc.jwtsecret=/home/ubuntu/key-hex/jwt.hex --batchSize 128m --downloader.verify=false
`
